README.rst

```diff
@@ -1,2 +1,54 @@
 This package provides an ExplainCollection class
-that allows common MongoDB commands to be explained
\ No newline at end of file
+that allows that allows PyMongo's Collection methods to be explained.
```

Comment:
Can you link "explained" to the MongoDB public documentation of the `explain` command?

Reply:
Done.